docs/Analyzer Configuration.md

```diff
@@ -65,23 +65,23 @@ Default Value: All output kinds
 Example: `dotnet_code_quality.CA2007.output_kind = ConsoleApplication, DynamicallyLinkedLibrary`
 
 ### Async void methods
-Option Name: `skip_async_void_methods`
+Option Name: `exclude_async_void_methods`
 
 Configurable Rules: [CA2007](../src/Microsoft.CodeQuality.Analyzers/Microsoft.CodeQuality.Analyzers.md#ca2007-do-not-directly-await-a-task)
```

Comment:
@gewarren We need a docs.microsoft.com page for this new rule CA2007 (did not exist in legacy FxCop).